docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -9,13 +9,13 @@ Install the 4.x version of the `@aws-amplify/ui-react` library.
 <TabItem title="npm">
 
 ```shell
-npm install aws-amplify @aws-amplify/ui-react@4.x
+npm install aws-amplify@latest @aws-amplify/ui-react@4.x
```

Comment:
What do you think of specifying the version, since this will be out of date in the next release?
```suggestion
npm install aws-amplify@5.x @aws-amplify/ui-react@4.x
```

Reply:
👍